server/athenian/api/internal/miners/jira/issue.py

```diff
@@ -888,6 +950,36 @@ def hint_epics(q):
     return df
 
 
+@sentry_span
+async def _query_jira_raw_by_keys(
+    keys: Sequence[str],
+    columns: list[InstrumentedAttribute],
+    jira_config: JIRAConfig,
+    mdb: Database,
+    cache: Optional[aiomcache.Client],
+) -> pd.DataFrame:
+    where = [
+        Issue.acc_id == jira_config.acc_id,
+        Issue.project_id.in_(jira_config.projects),
+        Issue.is_deleted.is_(False),
+        Issue.key.progressive_in(keys),
+    ]
+    issue_status_join = sa.outerjoin(
+        Issue,
+        Status,
+        sql.and_(Issue.status_id == Status.id, Issue.acc_id == Status.acc_id),
+    )
+    issue_athenian_issue_join = sa.outerjoin(
+        issue_status_join,
+        AthenianIssue,
+        sql.and_(Issue.acc_id == AthenianIssue.acc_id, Issue.id == AthenianIssue.id),
+    )
+    stmt = sa.select(*columns).select_from(issue_athenian_issue_join).where(*where)
+    stmt = stmt.with_statement_hint(f"Leading({Issue.__tablename__}, *VALUES*)")
```

Comment:
```suggestion
    if len(keys) > 100:
        stmt = stmt.with_statement_hint(f"Leading({Issue.__tablename__} *VALUES*)")
```
- condition
- removed the comma `,`